volatility3/framework/plugins/linux/pstree.py

```diff
@@ -72,6 +76,12 @@ def yield_processes(pid):
             task = self._tasks[pid]
 
             row = self._get_task_fields(task, decorate_com)
+            if dump:
+                file_output = self._get_file_output(task)
+            else:
+                file_output = "Disabled"
+            row = self._get_task_fields(task, decorate_com)
```

Comment:
According to CodeQL, `row` needs multiple definitions?

Reply:
yup! It is pretty silly of me. I'll get it sorted. Thanks.